ui/src/themes/spotify.js

```diff
@@ -311,6 +311,38 @@ export default {
       input: {
         paddingLeft: '.9rem',
         border: 0,
+        '& .MuiInputBase-root': {
+          backgroundColor: 'white !important',
+          borderRadius: '20px !important',
+          color: 'black',
+          border: '0px',
+          '& fieldset': {
+            borderColor: 'white',
+          },
+          '&:hover fieldset': {
+            borderColor: 'white',
+          },
+          '&.Mui-focused fieldset': {
+            borderColor: 'white',
+          },
+          '& svg': {
+            color: 'black !important',
+          },
+
+          '& .MuiOutlinedInput-input:-webkit-autofill': {
+            borderRadius: '20px 0px 0px 20px',
+            '-webkit-box-shadow': '0 0 0 100px #c2c1c2 inset',
+            '-webkit-text-fill-color': 'black',
+          },
+        },
+      },
+    },
+    RaFilter: {
+      form: {
+        '& .MuiOutlinedInput-input:-webkit-autofill': {
+          '-webkit-box-shadow': '0 0 0 100px #28282b inset',
+          '-webkit-text-fill-color': 'white',
+        },
```

Comment:
This gets rid of the obnoxious blue color we got when we autocomplete in filtered fields

![Screenshot from 2021-04-07 20-18-33](https://user-images.githubusercontent.com/54576074/113942835-28273b80-981f-11eb-925c-579fea2bde79.png)